### DISCUSSION TITLE
Question v6 Mac Safari

### DISCUSSION THREAD

**Discussion by crossbench** [katspaugh/wavesurfer.js#2827] [Q&A]
Thank you for using our services. I'm using version 6.

When playing several songs continuously on Mac Safari version 16.5 (18615.2.9.11.4) and Ventura 13.4, it consumes a large amount of memory, ranging from 1 to 6 GB. There are no issues with Chrome or Firefox.

Additionally, when reloading and playing the first song, it is not recognized as autoplay and cannot be played.

Error message:
Unhandled Promise Rejection: NotAllowedError: The request is not allowed by the user agent or the platform in the current context, possibly because the user denied permission.

Do you have any solutions for these issues?

**Comment by katspaugh** [comment]
I would suggest checking if you encounter these issues with v7.
